proto/xudrpc.proto

```diff
@@ -596,11 +608,19 @@ message PlaceOrderEvent {
   }
 }
 
+message RaidenChannels {
```

Comment:
Can we rename `LndChannels` to `Channels` and just use that single message?

Reply:
I'm commenting out setting `settled` for raiden then ?